### ISSUE TITLE
npm install from aion_solo_pool throwing error

### ISSUE THREAD

**Issue by JSorenSwanson** [aionnetwork/aion_miner#2]
npm install works appropriately in equihashverify (also within node-blake2 and stratum-pool) directory but throws "npm ERR! equihashverify@0.0.1's `bundledDependencies` should be an array" when executed from root aion_solo_pool dir. 

npm v 3.5.2, node v6.11.4, node-gyp v3.4.0. 
miner reference client compiles perfectly. 
![screenshot from 2018-02-03 00-24-32](https://user-images.githubusercontent.com/26172223/35763784-54748b86-0879-11e8-84b1-7dbe7478c421.png)

**Comment by aion-Ross** [comment]
Could you try upgrading your node install?

The node version should be 8.9.3+

**Comment by JSorenSwanson** [comment]
I had some time to sit down today and get everything configured. I ran into a few more problems involving npm/node/node-gyp and finally narrowed the problem down to there being a permissions error involving npm package installation. 

I fixed this by: 

- Making a directory for global installations 
`mkdir ~/.npm-global` 
- Configuring npm to use new path
`pm config set prefix '~/.npm-global'`
- Creating a ~/.profile file and adding PATH:
`export PATH=~/.npm-global/bin:$PATH`
` source ~/.profile`

After I configured this directory, I reinstalled all the listed aion_solo_pool dependencies (node-gyp, node, bignum, bindings, etc.) and then ran `sudo npm install --force` in the root aion_solo_pool folder. The shell now executes correctly. 

Linux Kernel v.4.15.0-041500, Ubuntu 17.10.
